virttest/test_setup.py

```diff
@@ -243,22 +244,20 @@ def __init__(self, test, params):
             base_dir = f[0]
             if f[2]:
                 for name in f[2]:
-                    f_dir = os.path.join(base_dir, name)
-                    with open(f_dir, 'r') as param_f:
-                        parameter = param_f.read()
-                    logging.debug("Reading path %s: %s", f_dir,
-                                  parameter.strip())
+                    f_dir = utils_misc.SysFS(os.path.join(base_dir, name),
```

Comment:
SysFS is in virttest/kernel_interface.py, not utils_misc.py.

Reply:
@huangyum Thank for for reviewing, I will fix it.

Reply:
fixed it.